packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -23,46 +23,75 @@ class DiffPaneController {
 
   /// The list contains one item that show information and all others
   /// are snapshots.
-  ValueListenable<List<DiffListItem>> get snapshots => _snapshots;
-  final _snapshots = ListValueNotifier(<DiffListItem>[InformationListItem()]);
+  ValueListenable<List<SnapshotItem>> get snapshots => _snapshots;
+  final _snapshots = ListValueNotifier(<SnapshotItem>[SnapshotDocItem()]);
 
-  ValueListenable<int> get selectedIndex => _selectedIndex;
-  final _selectedIndex = ValueNotifier<int>(0);
+  ValueListenable<int> get selectedSnapshotIndex => _selectedSnapshotIndex;
+  final _selectedSnapshotIndex = ValueNotifier<int>(0);
+  void setSelectedSnapshotIndex(int index) =>
+      _selectedSnapshotIndex.value = index;
 
   /// If true, some process is going on.
   ValueListenable<bool> get isProcessing => _isProcessing;
   final _isProcessing = ValueNotifier<bool>(false);
 
-  DiffListItem get selectedItem => snapshots.value[selectedIndex.value];
+  SnapshotItem get selectedSnapshotItem =>
+      snapshots.value[selectedSnapshotIndex.value];
 
   /// Full name for the selected class.
-  ValueListenable<String?> get selectedClass => _selectedClass;
-  final _selectedClass = ValueNotifier<String?>(null);
-  void setSelectedClass(String? value) => _selectedClass.value = value;
+  ValueListenable<HeapClassName?> get selectedClass => _selectedClass;
+  final _selectedClass = ValueNotifier<HeapClassName?>(null);
+  void setSelectedClass(HeapClassName? value) => _selectedClass.value = value;
```

Comment:
nit: selectedClassName

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-naming-the-argument-to-a-setter-value

Reply:
I'm referring to all three of getter-private var-setter here, not the `value` parameter.
get selectedClassName
_selectedClassName
set selectedClassName(HeapClassName? value (can we also use a real setter for this one instead of a method that has 'set' in its name?)